tests/ci_install.sh

```diff
@@ -12,34 +34,34 @@ if [[ "$TRAVIS_OS_NAME" == "windows" ]] && [[ $MINIMAL != 1 ]]; then
     H5PY=$(grep h5py tests/test_requirements.txt)
     MATPLOTLIB=$(grep matplotlib tests/test_requirements.txt)
     SCIPY=$(grep scipy tests/test_requirements.txt)
+    conda config --set always_yes yes
+    conda info -a
     conda install --yes -c conda-forge $CYTHON $NUMPY $CARTOPY $H5PY $MATPLOTLIB $SCIPY
 else
-    ccache -s
-    # Upgrade pip and setuptools and wheel to get clean install
-    $PIP install --upgrade pip
-    $PIP install --upgrade wheel
-    $PIP install --upgrade setuptools
+    python -m pip install --upgrade pip
+    python -m pip install --upgrade wheel
+    python -m pip install --upgrade setuptools
 fi
 
 # Step 2: install required packages (depending on whether the build is minimal)
-if [[ $MINIMAL == 1 ]]; then
+if [[ ${dependencies} == "minimal" ]]; then
     # Ensure numpy and cython are installed so dependencies that need to be built
     # don't error out
     # The first numpy to support py3.6 is 1.12, but numpy 1.13 matches
     # unyt so we'll match it here.
-    $PIP install numpy==1.13.3 cython==0.26.1
-    $PIP install -r tests/test_minimal_requirements.txt
+    python -m pip install numpy==1.13.3 cython==0.26.1
+    python -m pip install -r tests/test_minimal_requirements.txt
 else
     # Getting cartopy installed requires getting cython and numpy installed
     # first; this is potentially going to be fixed with the inclusion of
     # pyproject.toml in cartopy.
     # These versions are pinned, so we will need to update/remove them when
     # the hack is no longer necessary.
-    $PIP install -r tests/test_prerequirements.txt
-    CFLAGS="$CFLAGS -DACCEPT_USE_OF_DEPRECATED_PROJ_API_H" $PIP install -r tests/test_requirements.txt
+    python -m pip install -r tests/test_prerequirements.txt
+    CFLAGS="$CFLAGS -DACCEPT_USE_OF_DEPRECATED_PROJ_API_H" python -m pip install -r tests/test_requirements.txt
 fi
 
 # Step 3: install yt
-$PIP install -e .
+python -m pip install -e .
 
-set +x  # Do not show which command is being run (Travis default)
\ No newline at end of file
+set +x  # Do not show which command is being run (Travis default)
```

Comment:
Is this line still necessary ? Even if it is, the comment should be updated

Reply:
I don't know what's the default for GHA environment, but I don't think that setting it explicitly is a bad thing either. I'll drop the comment.